BUILDING.md

```diff
@@ -94,14 +94,14 @@ Depending on host platform, the selection of toolchains may vary.
 #### OpenSSL asm support
 
 OpenSSL-1.1.0 requires the following asssembler version for use of asm
-support.
+support on x86_64 and ia32.
 
 * gas (GNU assembler) version 2.23 or higher
 * xcode version 5.0 or higher
 * llvm version 3.3 or higher
 * nasm version 2.10 or higher in Windows
 
-Otherwise, `--openssl-no-asm` is added with warning in configure.
+Otherwise, configure is failed. Install one or build with --openssl-no-asm.
```

Comment:
`` configure `` -> `` `configure` `` and `` --openssl-no-asm `` -> `` `--openssl-no-asm` ``?

Reply:
@vsemozhetbyt Thanks and very sorry for I often missed backquotes. Fixed in 0388dbd9905c927ddef45e46bdc4d336a9164a8d.

Reply:
It is OK, cryptography domain is complicated enough to not also track all the doc nits)

Reply:
minor nit: 'Otherwise `configure` will fail with an error. This can be avoided by either providing a new enough assembler as per the list above or by using the `--openssl-no-asm` flag.'